presto-product-tests/src/main/java/com/facebook/presto/tests/hive/TestHiveCoercion.java

```diff
@@ -0,0 +1,354 @@
+/*
```

Comment:
Sure! Thanks for assigning it to me because I totally missed @geraint0923's @mention.